backend/tests/integration/records/indexer/test_indexer_literature.py

```diff
@@ -72,3 +68,49 @@ def test_indexer_deletes_record_from_es(es_clear, db, datadir, create_record):
     hits_total = response["hits"]["total"]
 
     assert hits_total == expected_total
+
+
+@patch("inspirehep.records.api.literature.LiteratureRecord.add_files")
+def test_indexer_creates_proper_fulltext_links_in_ui_display(
```

Comment:
we might need a test case when all the source fields are empty, and we don't return `fulltext_links` at all.

also tests for the detail view would be good to

Reply:
:+1: